### PR TITLE
Fixed association ids

### PR DESCRIPTION
**Pull request by Aristat**
Problem at the association id updating. I have made fix, if it not the correct behavior

### CODE REVIEW

**Comment by jkowens** [comment]
Can you give more details about what this is fixing?

**Comment by Aristat** [comment]
For example we have such structure - Topic and Books(topic_id)

When we have to change id of association
```
book.topic_id = new_topic_id
Book.import(book, on_duplicate_key_update: [:topic_id])
```
- it will not work, if the book exists, because https://github.com/zdennis/activerecord-import/blob/4d48b9ba4b690bb8a05e415f4c4a261c5b1f0abc/lib/activerecord-import/import.rb#L793 roll back the changes, will set old_topic_id

File: lib/activerecord-import/import.rb [hunk 1]
```diff
@@ -783,14 +783,17 @@ def set_attributes_and_mark_clean(models, import_result, timestamps, options)
 
     # Sync belongs_to association ids with foreign key field
     def load_association_ids(model)
+      changed_columns = model.changed
       association_reflections = model.class.reflect_on_all_associations(:belongs_to)
       association_reflections.each do |association_reflection|
+        column_name = association_reflection.foreign_key
         next if association_reflection.options[:polymorphic]
+        next if changed_columns.include?(column_name)
```
[the remark this answers is not in the document]
Comment (Aristat) [reply lib/activerecord-import/import.rb]: Such fix will correct this problem too :+1:

[the remark this answers is not in the document]
Comment (jkowens) [reply lib/activerecord-import/import.rb]: I think it's safe to assume if the foreign key is already populated, we can skip the assignment. Can you update the PR? I will get it merged once ready. Thanks!

[the remark this answers is not in the document]
Comment (Aristat) [reply lib/activerecord-import/import.rb]: Yes, I will update during weekend

[the remark this answers is not in the document]
Comment (Aristat) [reply lib/activerecord-import/import.rb]: @jkowens Fixed, `public_send("#{column_name}")` will not work if we dont have associations.
I corrected a code

**Comment by jkowens** [comment]
Thanks!

**Comment by jkowens** [comment]
Released in v0.24.0 👍

**Pull request merged**